gradle.properties

```diff
@@ -13,7 +13,7 @@ graphqlJavaToolsVersion=6.0.2
 graphqlSpqrVersion=0.10.1
 groovyVersion=2.5.7
 logbackClassicVersion=1.2.3
-micronautVersion=1.3.0
+micronautVersion=2.0.0.M3
```

Comment:
@alvarosanchez is this really needed for the `micronaut-graphql` `1.3` branch?
I would prefer keeping `micronaut` `1.3.x` as upgrading to `2.0` implies a major change, hence requiring `micronaut-graphql` `2.0` (`master`).

Reply:
I see 1.3.5 in the PR?

Reply:
I think even `1.3.0` on the branch itself https://github.com/micronaut-projects/micronaut-graphql/blob/1.3.x/gradle.properties#L16. 

So that is good!

I think I got confused by this:

![image](https://user-images.githubusercontent.com/197004/84008849-aeaa8c80-a972-11ea-8424-fc1d68dcfe9a.png)

Reply:
Hmm, I'm confused too with that commit being part of this PR. We have written some automations to sync files. It could have some glitches